src/js/plugin-bridge/PluginSDK.js

```diff
@@ -11,6 +11,7 @@ import Hooks from './Hooks';
 import PluginSDKStruct from './PluginSDKStruct';
 import Loader from './Loader';
 import PluginModules from './PluginModules';
+import RoutingService from '../../../foundation-ui/routing';
```

Comment:
Should we add `RoutingService` to `foundation-ui/index.js` so we can do `import {RoutingService} from 'foundation-ui';`?